ocaml5_tutorial.opam

```diff
@@ -12,7 +12,6 @@ depends: [
   "ocaml"
   "dune" {>= "3.1"}
   "odoc" {with-doc}
-  "domainslib"
```

Comment:
This should stay and be added to the `dune-project` file https://github.com/kayceesrk/ocaml5-tutorial/blob/main/dune-project#L22

Reply:
Thanks! You're right, will update it. It probably got automatically removed since it's not present in the dune-project file.